sample/src/main/java/vehicles/AbstractVehicle.java

```diff
@@ -0,0 +1,40 @@
+package vehicles;
+
+import people.Person;
+
+import java.time.LocalDate;
+
+/**
+ * An abstract class for implementing different kinds of {@link Vehicle}.
```

Comment:
Apparently, any other javadoc tag such as @author, @param, @return will  not be parsed.
Such tags may be converted to asciidoc tags.
What I'm thinking it to make the exportdoclet to parse such javadoc tags and convert them to a asciidoc one.